test/PerformanceTests/SerializationComparisonsTests/JsonWriterBenchmarks/Benchmarks.cs

```diff
@@ -0,0 +1,81 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="Benchmarks.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+using BenchmarkDotNet.Diagnosers;
+using ExperimentsLib;
+using Microsoft.OData.Edm;
+
+namespace JsonWriterBenchmarks
+{
+    [MemoryDiagnoser]
+    [Orderer(SummaryOrderPolicy.Method, MethodOrderPolicy.Declared)]
+    public class Benchmarks
+    {
+        IEnumerable<Customer> data;
+        IEdmModel model;
+
+        public Stream memoryStream;
+        public Stream fileStream;

```

Comment:
Public fields? Why? Should they be in PascalCase?

Reply:
Should be private. Thanks.